shell/server/server-middleware.js

```diff
@@ -1,11 +1,9 @@
-import { URL } from 'url';
-
-export default function(req, res, next) {
-  const parsed = new URL(req.url, 'https://localhost');
+module.exports = function(req, res, next) {
```

Comment:
We have to switch to this module system to make `require()` work in vue.config.js